### PR TITLE
[core] Move SmallPacket0x04B server message logic into Lua

### LINKED ISSUE #3965: 🔨 move SmallPacket0x04B handling to lua
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Describe the feature
As more and more gets stuffed here to handle messaging that happens at player login time  
https://github.com/LandSandBoat/server/pull/3933/files#diff-5c1417f13de5d4c90301639e386e3bec2fd53753b62c4aab5563f9c2d46c28d4R2294  
it is getting more and more apparent that there is a need to store the conditional logic and text outside of this part of the core in a more convenient manner. The issue lua side is timing those messages to reach the screen (the packets can be sent before the player has fully connected). People have worked around this with use of `timer()` before but that is not ideal, as people have varying latency and what is slow enough for one person isn't for the next, or is even too slow to look like retail at all.

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Closes https://github.com/LandSandBoat/server/issues/3965

Also removed deprecated/unused SetDropRate & UpdateServerMessage

## Steps to test these changes

image

### CODE REVIEW

**Comment by zach2good** [comment]
Out of draft, ready for review

File: scripts/globals/server.lua [hunk 1]
```diff
@@ -16,4 +16,23 @@ end
 xi.server.onTimeServerTick = function()
 end
 
+-- Message for use with SmallPacket0x04B
+xi.server.getServerMessage = function(language)
+    local serverMessage = ""
+
+    if language == 0x02 then -- English
```
Comment (claywar) [inline scripts/globals/server.lua]: Just a thought for the future, but should we eventually create an enum for language?

**Review by claywar — APPROVED** [review]

File: scripts/globals/server.lua (hunk 1 above)
Comment (zach2good) [reply scripts/globals/server.lua]: Good idea, added now

**Comment by zach2good** [comment]
Interesting added aside: I think now the server messages are hot-reloadable because they're just reading from settings. Neat.

**Pull request merged**